### PR TITLE
E_CLASSROOM-44 [Frontend] [Student] [Markup] User Profile 1/3

### PR DESCRIPTION
**Pull request by ErickDoroy**
### Issue Link
- https://framgiaph.backlog.com/view/E_CLASSROOM-44

### Definition of Done
- [x] Create markup page when viewing user profile.

#### Main note
- http://localhost:3003/profile
#### Pages Affected
- 

### Scenarios/ Test Cases
- [x] Can view user profile.  
#### User Interface
- N/A
#### User Experience 
- N/A
#### Functionality
- N/A
### Screenshot
Profile
s

### CODE REVIEW

**Comment by geneno99** [comment]
### Comments

- [ ] Please check the size of the avatar not perfect circle
- [ ] The two edit button is seems smaller but in figma is larger please checj 🙇🏼 

Screen Shot 2021-11-03 at 5 42 10 PM

-----
- Test time: 0.5 hr

**Comment by geneno99** [comment]
### Approved PR #12 

- [X] Able to implement User Profile look same as styling, position, and aligment as FIGMA

Screen Shot 2021-11-03 at 6 33 54 PM

-----
- Total Test time: 0.75 hr
- Total Iterations: 1
- Total Comments: 1

File: src/pages/student/profile/ProfileDetail/index.module.css [hunk 1]
```diff
@@ -7,52 +7,93 @@
     height: 418px;
     left: 450px;
     top: 170px;
-
+}
+body{
```
Comment (erronerron) [inline src/pages/student/profile/ProfileDetail/index.module.css:11]: Remove body styling

**Comment by erronerron** [comment]
- [ ] Fix conflicts

**Pull request merged**